modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {accessibilityMenuPagesTest} from '../../fixtures/accessibilityMenuPagesTest';
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {instanceSettingsPagesTest} from '../../fixtures/instanceSettingsPagesTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {usersAndOrganizationsPagesTest} from '../../fixtures/usersAndOrganizationsPagesTest';
+import performLogin, {performLogout, userData} from '../../utils/performLogin';
+import {featureFlagPagesTest} from '../feature-flag-web/fixtures/featureFlagPagesTest';
+
+const test = mergeTests(
+	accessibilityMenuPagesTest,
+	apiHelpersTest,
+	featureFlagPagesTest,
+	instanceSettingsPagesTest,
+	isolatedSiteTest,
+	loginTest(),
+	usersAndOrganizationsPagesTest
+);
+
+test('Enable accessibility menu', async ({
+	accessibilityMenuPage,
+	instanceSettingsPage,
+}) => {
+	await instanceSettingsPage.goToInstanceSetting(
+		'Accessibility',
+		'Accessibility Menu'
+	);
+
+	await accessibilityMenuPage.enableAccessibilityMenu();
+});
+
+test('Verify that the default value is displayed when the user has never changed the accessibility setting, regardless of sign-in status.', async ({
+	accessibilityMenuPage,
+	apiHelpers,
+	page,
+	site,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount.alternateName] = {
+		name: userAccount.givenName,
+		password: 'test',
+		surname: userAccount.familyName,
+	};
+
+	await test.step('Verify that the underlined links toggle is off when logged out, then turn it on', async () => {
+		await performLogout(page);
+
+		await page.goto(site.friendlyUrlPath);
+
+		await accessibilityMenuPage.openAccessibilityMenu();
+
+		await accessibilityMenuPage.toggleUnderlinedLinks(true);
```

Comment:
This method won't assert that the toggle is off to start with. So in this test we need to check the default value before toggling.